src/Teams/Services/ManageTeamsMembersService.cs

```diff
@@ -16,9 +17,26 @@ public class ManageTeamsMembersService : IManageTeamsMembersService
         private readonly IRepository<Team, int> _teamRepository;
         private readonly IRepository<TeamMember, int> _teamMemberRepository;
 
-        public ManageTeamsMembersService(IRepository<TeamMember, int> teamMemberRepository)
+        public ManageTeamsMembersService(IRepository<Team, int> teamRepository, IRepository<TeamMember, int> memberRepository, ICurrentUser currentUser)
+        {
+            _currentUser = currentUser;
+            _teamRepository = teamRepository;
+            _teamMemberRepository = memberRepository;
+        }
+        
+        public async Task<bool> RemoveAsync(int team_id, string member_id)
+        {
+            var team = await _teamRepository.GetAll().Where(t => t.Id == team_id).FirstOrDefaultAsync();
```

Comment:
Did you check this code block? It's seems that the team members will not be loaded because of the missing Include extension method call.